packages/protocol/contracts/team/airdrop/ERC20Airdrop.sol

```diff
@@ -43,6 +46,6 @@ contract ERC20Airdrop is MerkleClaimable {
 
     function _claimWithData(bytes calldata data) internal override {
         (address user, uint256 amount) = abi.decode(data, (address, uint256));
-        IERC20Upgradeable(token).transferFrom(vault, user, amount);
```

Comment:
No need to use it here because it's only for TKO (and TKO follows the ERC20 standard correctly)?